### ISSUE TITLE
Pre-trained image model loading

### ISSUE THREAD

**Issue by Z-Lee-corder** [PJLab-ADG/LoGoNet#30]
Hi, I found line 17  "IMGPRETRAINED_MODEL: '../output_s2/det_model_cfgs/pretrained_ckpt/swin_tiny_patch4_window7_224.pth'" in LoGoNet-kitti.yaml has been commented out

But I did not find any loss in the loss function regarding the image. This means that the image network is a pre-trained model. 
Therefore, should line 17 be uncommented？

In addition，I didn't understand line 9-10
    # remember to use "--pretrained_model" to load 1-stage checkpoint
    # and use "--fix_pretrained_weights" to fix params

Should these also be changed, if using pre-rained image models?

In short, I would like to use the pre-trained image model you provided. Which configurations should I change in the project?

**Comment by fangweicheng6** [comment]
Hi, have you solved your problem? I have the same problem.
